Boot.st

```diff
@@ -83,8 +83,7 @@ devsesh onPreSaveImage.
 	maxObjects: 24576*1024
 ] ensure: [devsesh onPostSaveImage]!
 
-"Remove rogue .img and unnecessary .chg files"
-File delete: (File default: SessionManager current imagePath extension: 'img').
```

Comment:
This doesn't seem quite right. "SessionManager current" will be an instance of the DevelopmentSessionManager at this point.
Do we know what is causing this "rogue" image to be saved in the first place?